docs/source/guides/contribution_guide.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+![NuCypher Unicorn](https://cdn-images-1.medium.com/max/800/1*J31AEMsTP6o_E5QOohn0Hw.png)
+
+
+## Running the Tests
+
+``` note:: A development installation including the solidity comppiler is required to run the tests
+```
+
+There are several test implementations in `nucypher`; However, the vast majority
```

Comment:
```suggestion
There are several test implementations in `nucypher`, however, the vast majority
```

Reply:
Heh - have I been using "However" wrong this whole time ?!? :-) <https://web.sonoma.edu/users/f/farahman/subpages/utilities/however.pdf>